topic.go

```diff
@@ -48,6 +48,7 @@ type partition struct {
 	NewestOffset int64   `json:"newest"`
 	Leader       string  `json:"leader,omitempty"`
 	Replicas     []int32 `json:"replicas,omitempty"`
+	Isrs         []int32 `json:"isrs,omitempty"`
```

Comment:
@pd could you rename this to ISRs? cf https://github.com/golang/go/wiki/CodeReviewComments#initialisms